bundle/edu.gemini.phase2.skeleton.servlet/src/main/scala/edu/gemini/phase2/skeleton/factory/SpProgramFactory.scala

```diff
@@ -179,47 +179,42 @@ object SpProgramFactory {
     itacAcceptance(proposal).exists(_.rollover)
 
   def isThesis(proposal: Proposal): Boolean =
-    proposal.investigators.all exists { i => i.status == GRAD_THESIS }
+    proposal.investigators.all.exists(_.status == GRAD_THESIS)
 
   def gemEmail(proposal: Proposal): Option[String] =
     itacAcceptance(proposal).flatMap(_.contact)
 
   def minBand3Time(proposal: Proposal): Option[TimeValue] =
     proposal.proposalClass match {
-      case q: QueueProposalClass => q.band3request map { r => toTimeValue(r.minTime) }
+      case q: QueueProposalClass => q.band3request.map(r => toTimeValue(r.minTime, proposal.programTimeRatio))
       case _                     => None
     }
 
-  private def toTimeValue(ta: TimeAmount): TimeValue =
+  private def toTimeValue(ta: TimeAmount, ratio: Double = 1.0): TimeValue =
     ta.units match {
-      case TimeUnit.NIGHT => new TimeValue(ta.value, TimeValue.Units.nights)
-      case _              => new TimeValue(ta.toHours.value, TimeValue.Units.hours)
+      case TimeUnit.NIGHT => new TimeValue(ta.value * ratio, TimeValue.Units.nights)
+      case _              => new TimeValue(ta.toHours.value * ratio, TimeValue.Units.hours)
     }
 
   def timeAcctAllocation(proposal: Proposal): Option[TimeAcctAllocation] =
-    awardedHours(proposal).filter(_ > 0.0) flatMap { hrs =>
-      val progTime = hoursFromObservations(proposal, _.progTime)
-      val partTime = hoursFromObservations(proposal, _.partTime)
-
+    // We ignore the awarded hours here and work instead with the program time and partner time.
+    awardedHours(proposal).filter(_ > 0.0) flatMap { _ =>
       timeAccountingRatios(proposal) match {
         case Nil => None
         case ratios =>
           val jmap = ratios.map { case (cat, rat) =>
             def durationRatio(v: TimeAmount): Duration =
               Duration.ofMillis(((v.hours * rat) * MsPerHour).round)
 
-            val award = new TimeAcctAward(durationRatio(progTime), durationRatio(partTime))
+            val award = new TimeAcctAward(durationRatio(proposal.programTime), durationRatio(proposal.partnerTime))
```

Comment:
> Yes, that's right. We determined that it should always be the case that awarded time = sum over observations (progTime + partTime), so we could maintain things as they were.

I'm probably being overly pessimistic but I'd feel better if the skeleton creation and the itac email were based on the same total award value taken from the same place in the model.  Then there's zero chance that we come up with different answers.  Something like:

```scala
awardedHours(proposal).filter(_ > 0.0).flatMap { hrs =>
...
  case ratios =>
    val jmap = ratios.map { case (cat, partnerShare) =>
      def duration(rat: Double): Duration =
        Duration.ofMillis((hrs.hours * rat * partnerShare) * MsPerHour).round)

      val award = new TimeAcctAward(
                    duration(proposal.programTimeRatio),
                    duration(1.0 - proposal.programTimeRatio))
      (cat, award)
    }.toMap.asJava
...
}
```